website/.docusaurus/client-modules.js

```diff
@@ -0,0 +1,3 @@
+export default [
+  require("infima/dist/css/default/default.css"),
```

Comment:
this files should be git ignored. Dont we have a “.docusaurus” ignore on gitignore